code/modules/crafting/table.dm

```diff
@@ -98,6 +101,8 @@
 						I.reagents = new /datum/reagents()
 					I.reagents.reagent_list.Add(A)
 			I.CheckParts()
+			if(send_feedback)
+				feedback_add_details("object_crafted","[I.name]")
```

Comment:
Recommend you replace with type or some other identifier instead of name, name is often changed by admins or code